src/main/groovy/com/palantir/gradle/docker/PalantirDockerPlugin.groovy

```diff
@@ -181,6 +181,9 @@ class PalantirDockerPlugin implements Plugin<Project> {
             if (ext.load) {
                 buildCommandLine.add '--load'
             }
+	    if (ext.push) {
+	        buildCommandLine.add '--push'
```

Comment:
note that `docker buildx build` does not allow `--load` and `--push` to be combined. Might be worth adding something like:
```
if (ext.load) {
    throw new RuntimeError("cannot combine 'push' and 'load' options");
}
```
to make it clear.